build/xtask/src/dist.rs

```diff
@@ -573,6 +575,66 @@ fn write_gdb_script(cfg: &PackageConfig, image_name: &str) -> Result<()> {
     Ok(())
 }
 
+/// Encode the given data as a tagged TLV-C chunk
+fn data_to_tlvc(tag: &str, data: &[u8]) -> Result<Vec<u8>> {
+    if tag.len() != 4 {
+        bail!("Tag must be a 4-byte value, not '{}'", tag);
+    }
+    let mut out = vec![];
+    let mut header = tlvc::ChunkHeader {
+        tag: tag.bytes().collect::<Vec<u8>>().try_into().unwrap(),
```

Comment:
I think this could be 
```suggestion
        tag: tag.as_bytes().try_into().unwrap(),
```